src/test/java/ParserTest.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Test class to test parse() method in Parser.
+ */
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void parseTest() {
+        assertEquals(new DoneCommand(2), Parser.parse("done 3"));
```

Comment:
Could I ask why is the command DoneCommand(2) instead of DoneCommand(3) ? As the attribute name in constructor DoneCommand is called taskNum. Thank you!